pkgs/development/node-packages/default.nix

```diff
@@ -8,49 +8,35 @@ let
     inherit pkgs nodejs;
     inherit (stdenv.hostPlatform) system;
   };
-  self = super // {
+  addMainPrograms = lib.mapAttrs (pkgName: mainProgram:
+    super.${pkgName}.override (oldAttrs: {
+      meta = oldAttrs.meta // { inherit mainProgram; };
+    })
```

Comment:
Can you merge this with the other commit to make the history less noisy?

Reply:
Sure!

Reply:
Done.